languagetool-language-modules/el/src/main/resources/org/languagetool/resource/el/specific_case.txt

```diff
@@ -0,0 +1,76 @@
+# this file was based on extraction from the Wikipedia corpus
```

Comment:
Maybe specify the exact name of the class that was used to generate this file.

Reply:
I added the path to the corresponding class 9b47ef46c13fac1fc04aa18b6066851baefc1ee6